example/app-router/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "app-router",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```

Comment:
add the following preinstall script
`"preinstall": "cd ../../ && npm install && npm pack"`

Reply:
done